spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
+    before :each do
+      Division.delete_all
 
+      @division = Division.create(id: 5, name: "hey jude", date: Date.new(2016,12,25),
+      number: 1, house: "representatives", source_url: "", debate_url: "", motion: "",
+      source_gid: "", debate_gid: "")
+
+      @other_division = Division.create(id: 6, name: "beautiful boy", date: Date.new(2016,06,01),
+      number: 1, house: "representatives", source_url: "", debate_url: "", motion: "",
+      source_gid: "", debate_gid: "")
+    end
+
+    context "when request has no parameters" do
+      it "should return index page with all divisions" do
```

Comment:
"with all divisions" I don't think this is actually true. If you load https://theyvoteforyou.org.au/divisions/ it seems to include divisions for the latest calendar year that we have divisions for.